assembly/std/arguments.ts

```diff
@@ -0,0 +1,165 @@
+import {Address} from './address';
+import {ByteArray} from '@massalabs/as/assembly/byteArray';
+
+/**
+ * Args for remote function call
+ */
+export class Args {
+  private offset: i64 = 0;
+  private argsString: string = '';
+
+  /**
+   *
+   * @param {string} argsString
+   * @param {Array<string>} types
+   */
+  constructor(argsString: string = '') {
```

Comment:
This argument is misleading : you reuse the name of the package and the name of the type.

Using something like `serialized` will make the constructor clearer:

```typescript
var args = new Args(serialized);
```